### DISCUSSION TITLE
Unexpected Exception: The GET method is not supported for route livewire/update. Supported methods: POST.

### DISCUSSION THREAD

**Discussion by aimonext** [filamentphp/filament#11854] [Help]
### Package

Form Builder

### Package Version

v3.0.0

### This exception has occurred since 3 days. This is just a simple custom form in a livewire component. In local this works smoothly but the problem occurs in production hosting.
![Screenshot from 2024-03-15 00-00-59](https://github.com/filamentphp/filament/assets/110348549/ae852fcc-a339-4622-85fd-9a4d0726f9f7)
![Screenshot from 2024-03-15 00-01-07](https://github.com/filamentphp/filament/assets/110348549/a198a712-6d4b-4b13-8c42-714d85c6ee00)

**Comment by leandrocfe** [comment]
There are some discord topics about this, and t’s connected to your server configuration for using Livewire.
https://discord.com/channels/883083792112300104/1217382026160902256/1217382026160902256
